docs/release-notes/release-notes-0.14.2.md

```diff
@@ -44,6 +48,9 @@
 # Contributors (Alphabetical Order)
 
 * Andras Banki-Horvath
+* Bjarne Magnussen
+* Harsha Goli
```

Comment:
Could just squash this commit with the last one so it doesn't look like you're removing then adding people again to the contributor list.